src/Website/docs/JSON.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Dealing with JSON
+subtitle: Use the inbuilt extensions to [System.Text.Json](https://github.com/Tarmil/FSharp.SystemTextJson) for handling F# union and records
```

Comment:
Can you make the link to `FSharp.SystemTextJson` here be on the term "extensions" and not `System.Text.Json`? Also, you might want to make "union" in the last bit a plural, or just use the more encompassing but somewhat dull "F#-specific types" instead.

Reply:
done